### ISSUE TITLE
Metadata reload functionality missing in pysaml 7.0.1

### ISSUE THREAD

**Issue by vladimir-mencl-eresearch** [IdentityPython/SATOSA#385]
Hi @c00kiemon5ter , great to see version 8.0.0 out!

I see the release notes mention the new `/reload-metadata` endpoints - and give `pysaml2 > 7.0.1` as the requirement.

Unfortunately, this functionality was merged into pysaml2 only *after* 7.0.1 was released (in IdentityPython/pysaml2#809) - and there has been no pysaml2 release since.

This could be solved with a pysaml2 release - and an update to the release notes for SATOSA 8.0.0...

Cheers,
Vlad

## Code Version
SATOSA  8.0.0
pysaml2 7.0.1

## Expected Behavior

Requests to `/reload-metadata` URLs working with when `enable_metadata_reload: yes` is in the config.

## Current Behavior

Requests to `/reload-metadata` URLs failing on invocation of reload_metadata.

## Possible Solution

Releasing pysaml2 with the functionality added in IdentityPython/pysaml2#809

## Steps to Reproduce
1. Deploy SATOSA with SAML2 backend
2. Add `enable_metadata_reload: yes` into configuration
3. Issue a request to `/Saml2/reload-metadata`
4. Request fails and a stack trace is produced.

**Comment by c00kiemon5ter** [comment]
> I see the release notes mention the new /reload-metadata endpoints - and give pysaml2 > 7.0.1 as the requirement.
>
> Unfortunately, this functionality was merged into pysaml2 only after 7.0.1 was released (in IdentityPython/pysaml2#809) - and there has been no pysaml2 release since.

that is correct; in the changelog we have (as you also write) `pysaml2 > 7.0.1` - not `pysaml2 >= 7.0.1`. So, we point to a newer than 7.0.1 pysaml2 version.

> This could be solved with a pysaml2 release

exactly! the changelog note is provisional at the moment. The intention was to have a pysaml2 release very close to the release of satosa v8.0.0. I didn't get the time to actually do this though, but I am working on it.

**Comment by vladimir-mencl-eresearch** [comment]
> that is correct; in the changelog we have (as you also write) `pysaml2 > 7.0.1` - not `pysaml2 >= 7.0.1`. So, we point to a newer than 7.0.1 pysaml2 version.

Sorry, completely missed that - yes, that is correct, I misread the release notes.

> exactly! the changelog note is provisional at the moment. The intention was to have a pysaml2 release very close to the release of satosa v8.0.0. I didn't get the time to actually do this though, but I am working on it.

Great - I look forward to seeing it out.

**Comment by c00kiemon5ter** [comment]
Pysaml2 `v7.1.0` is now available
